cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocClient.java

```diff
@@ -1088,4 +1141,9 @@ private RetryableProvisionException handleApiException(ApiException e) throws Re
       this.port = port;
     }
   }
+
+  private static Set<String> errorInfoReasons = Stream.of(
```

Comment:
Use immutable set for any static members. Also use uppercase underscored names for constants and move to the beginning of the class